manifests/prospector.pp

```diff
@@ -25,12 +32,17 @@
 
   validate_hash($fields, $multiline)
   validate_array($paths, $exclude_files, $include_lines, $exclude_lines, $tags)
+  validate_bool($tail_files, $close_renamed, $close_removed, $close_eof, $clean_removed)
 
   $prospector_template = $filebeat::real_version ? {
     '1'     => 'prospector1.yml.erb',
     default => 'prospector5.yml.erb',
   }
 
+  if $close_removed == false and $clean_removed == true {
+    notify('If you set close_removed to false, does not make sense to set clean_removed to true. Review the official Filebeat documentation')
```

Comment:
I'm not sure how much commentary to add about configuration issues - there are a number of conflicting configuration parameters that can be set (e.g. `ignore_older` has to be greater than `close_inactive`).

Checking and warning for every conflict would be difficult to maintain, so I'm inclined to leave them out entirely.